test/examples/OnChainERC721.test.js

```diff
@@ -0,0 +1,78 @@
+const encodeCall = require('zos-lib/lib/helpers/encodeCall').default;
+const { assertRevert } = require('../helpers/assertRevert');
+
+const BigNumber = web3.BigNumber;
+
+const should = require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+const OnChainERC721 = artifacts.require('OnChainERC721');
+
+contract.only('OnChainERC721', function ([_, deployer, minterA, minterB, pauserA, pauserB, anyone, ...otherAccounts]) {
```

Comment:
We need to add a check that makes the CI fail if there is an `only` statement

Reply:
At least we'll see the coverage drop abruptly :stuck_out_tongue: My bad!